src/common/sockman.h

```diff
@@ -133,6 +133,22 @@ class SockMan
     // alone to use the default implementation from SockMan.
     //
 
+    /**
+     * SockMan would only call EventReadyToSend() if this returns true.
```

Comment:
dc6393cb93c4851a363b69fd474656cac1ae3b3b: `EventReadyToSend` doesn't exist yet in this commit, so maybe the commit message can announce it.